argocd-notifications/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Argo CD Notifications container
+
+FROM quay.io/cybozu/golang:1.17-focal as builder
+
+ENV REPO=argoproj-labs/argocd-notifications
+ENV VERSION=1.2.1
+
+RUN apt-get update && apt-get install ca-certificates
+
+WORKDIR /work
+RUN curl -sSLf "https://github.com/${REPO}/archive/v${VERSION}.tar.gz" | \
+    tar --strip-components=1 -zxf -
+RUN go mod download
+RUN CGO_ENABLED=0 go build -ldflags="-w -s" -o /app/argocd-notifications ./cmd
+RUN ln -s /app/argocd-notifications /app/argocd-notifications-backend
```

Comment:
This creates a symbolic link in the builder container, but the COPY below creates a regular file.
This would not be what you want to do.

Executing `RUN ln -s` in the target container should work well.